HoneyBadgers.ConsoleApp/Test.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using HoneyBadgers.Logic;
+
+namespace HoneyBadgers.ConsoleApp
+{
+    public class Test
```

Comment:
nie jestem przekonana co do nazwy Test, do czego ma słuzyć ta klasa?

Reply:
rozumiem, u mnie w pracy kolega powiedziałby że to śmiecenie 😜 więc jeśli będziesz pamiętał żeby później usunąć to ok